functionary/core/models/package.py

```diff
@@ -55,6 +62,9 @@ class Package(models.Model):
 
     image_name = models.CharField(max_length=256)
 
+    objects = models.Manager()
```

Comment:
Does this need to be defined?  Seeing it here might make one assume that we're changing it away from its default, but that's not actually the case.